### PR TITLE
Fix bugs

### LINKED ISSUE #4359: fix up gitcoin.co/ens
- fix any obvious UI issues (like this footer http://bits.owocki.com/ef598bbe1162/Image%202019-05-03%20at%2011.09.51%20AM.png)
- update for any ENS changes that are live right now as ENS updates ( see https://medium.com/the-ethereum-name-service/upcoming-changes-to-the-ens-root-a1b78fd52b38 )

### PR DESCRIPTION
**Pull request by iamonuwa**
##### Description

Remove one of the metamask approval scripts to allow one error div show.
Fix footer style.

##### Refers/Fixes
Fixes #4359 

##### Testing

No tests on this one. This is a UI fix

### CODE REVIEW

**Comment by iamonuwa** [comment]
@owocki please take a look

**Comment by iamonuwa** [comment]
> @iamonuwa could you add a recording please?

Sure

File: app/assets/onepager/css/main.css [hunk 1]
```diff
@@ -560,6 +560,18 @@ body #main a:not(.button):hover {
...
+.alert {
+  position: absolute;
+}
+#footer {
+  position: absolute;
+  background-color: #0D0764;
+  padding: 5px;
+  width: 100vw;
+  left: 0px;
+  text-align: center;
```
Comment (thelostone-mc) [inline app/assets/onepager/css/main.css]: you could add `text-center` as opposed to redefining the style

File: app/assets/onepager/css/main.css [hunk 2]
```diff
@@ -560,6 +560,18 @@ body #main a:not(.button):hover {
   margin: auto;
 }
 
+.alert {
+  position: absolute;
+}
+#footer {
+  position: absolute;
```
Comment (thelostone-mc) [inline app/assets/onepager/css/main.css]: you could add `position-absolute` as opposed to redefining the style

File: app/assets/onepager/css/main.css [hunk 3]
```diff
@@ -560,6 +560,18 @@ body #main a:not(.button):hover {
   margin: auto;
 }
 
+.alert {
+  position: absolute;
```
Comment (thelostone-mc) [inline app/assets/onepager/css/main.css]: what is this style for ? Is this needed ?

File: app/assets/onepager/css/main.css [hunk 4]
```diff
@@ -560,6 +560,18 @@ body #main a:not(.button):hover {
...
+}
+#footer {
```
Comment (thelostone-mc) [inline app/assets/onepager/css/main.css]: Could you add an empty line between styles ?

File: app/dashboard/templates/ens/base.html [hunk 5]
```diff
@@ -56,14 +55,10 @@
...
-          <li>
-            <a href="/">&copy; {% trans "Gitcoin.co" %}</a>
-          </li>
-          <li>
-            {% blocktrans %}Made with <3 &amp; bounties by <a href="https://github.com/gitcoinco/web/pull/509" target="_blank" rel="noopener noreferrer" class="pull-request-link">Scottydelta and Gitcoin Core</a>{% endblocktrans %}
-          </li>
-        </ul>
+        <div class="copyright">
+          <a href="/">&copy; {% trans "Gitcoin.co" %}</a>
+            &nbsp;&nbsp;{% blocktrans %} Made with <3 &amp; bounties by <a href="https://github.com/gitcoinco/web/pull/509" target="_blank" rel="noopener noreferrer" class="pull-request-link">Scottydelta and Gitcoin Core</a>{% endblocktrans %}
```
Comment (thelostone-mc) [inline app/dashboard/templates/ens/base.html]: could you break this into 2 lines?

**Comment by iamonuwa** [comment]
@thelostone-mc 

Screenshot from 2019-05-14 08-54-32

File: app/assets/onepager/css/main.css (hunk 3 above)
Comment (octavioamu) [inline app/assets/onepager/css/main.css]: why this one? is not already absolute positioned?

File: app/assets/onepager/css/main.css [hunk 6]
```diff
@@ -560,6 +560,18 @@ body #main a:not(.button):hover {
   margin: auto;
 }
 
+.alert {
+  position: absolute;
+}
+
+#footer {
+  position: absolute;
+  background-color: #0D0764;
```
Comment (octavioamu) [inline app/assets/onepager/css/main.css]: for real center the content of this we can do 
left: 0;  right: 0;  text-align: center
In that way not matter the strength of the content will always be centered

**Comment by thelostone-mc** [comment]
@iamonuwa if you could address @octavioamu comments -> we can get this in

File: app/assets/onepager/css/main.css (hunk 3 above)
Comment (iamonuwa) [reply app/assets/onepager/css/main.css]: Not sure am the one that introduced this. But am going to remove it now

**Comment by iamonuwa** [comment]
@thelostone-mc @octavioamu fixed

File: app/assets/onepager/css/main.css (hunk 2 above)
Comment (thelostone-mc) [reply app/assets/onepager/css/main.css]: ^ this still hasn't been addressed

File: app/assets/onepager/css/main.css (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/onepager/css/main.css]: ^ this still hasn't been addressed

File: app/assets/onepager/css/main.css (hunk 3 above)
Comment (thelostone-mc) [reply app/assets/onepager/css/main.css]: you could add position-absolute as opposed to redefining the style

**Comment by thelostone-mc** [comment]
@iamonuwa seems like you've missed two of my comments 😅 
We wanna try to reuse css classes as much as possible. Hence the reason for enforcing it

File: app/assets/onepager/css/main.css (hunk 2 above)
Comment (iamonuwa) [reply app/assets/onepager/css/main.css]: Fixed

**Comment by iamonuwa** [comment]
Fixed

File: app/assets/onepager/css/main.css [hunk 7]
```diff
@@ -560,6 +560,19 @@ body #main a:not(.button):hover {
   margin: auto;
 }
 
+.alert {
+  position: absolute;
```
Comment (thelostone-mc) [inline app/assets/onepager/css/main.css]: @iamonuwa this can be removed as you've already added the style class

**Comment by owocki** [comment]
can we pls fix the dark font on the footer? http://bits.owocki.com/c4f421cfeb3d/Image%202019-05-21%20at%205.24.16%20PM.png

File: app/dashboard/templates/ens/base.html [hunk 8]
```diff
@@ -25,7 +25,6 @@
     <link rel="stylesheet" href="{% static "cookielaw/css/cookielaw.css" %}">
 
     <script src="{% static "v2/js/jquery.js" %}"></script>
-    <script src="{% static "v2/js/metamask-approval.js" %}"></script>
```
Comment (danlipert) [inline app/dashboard/templates/ens/base.html]: Why was this js script removed?

Comment (thelostone-mc) [inline app/dashboard/templates/ens/base.html]: @iamonuwa why was this removed? 
cc @gitcoinco/engineers

Comment (iamonuwa) [reply app/dashboard/templates/ens/base.html]: It introduces the second alert box.

**Comment by iamonuwa** [comment]
Still waiting for remaining reviews if any here. @danlipert @octavioamu @thelostone-mc

**Comment by iamonuwa** [comment]
@octavioamu @danlipert @thelostone-mc waiting for updates if any.

File: app/assets/onepager/css/main.css (hunk 3 above)
Comment (octavioamu) [reply app/assets/onepager/css/main.css]: pleaser remove this alert rules are already in base https://gitcoin.co/tip/send/2 you can check there if you disable metamask is fixed on top and also have rules to handle multiple alerts

File: app/assets/onepager/css/main.css (hunk 6 above)
Comment (octavioamu) [reply app/assets/onepager/css/main.css]: I keep seeing 100vw here not sure what do you want to solve here @iamonuwa 
What is already on production is perfectly centered and why putting the bg blue of this footer if is actually transparent?
image

File: app/dashboard/templates/ens/base.html [hunk 9]
```diff
@@ -24,8 +24,7 @@
     <link rel="stylesheet" href="{% static "v2/css/jquery.select2.min.css" %}">
     <link rel="stylesheet" href="{% static "cookielaw/css/cookielaw.css" %}">
 
-    <script src="{% static "v2/js/lib/jquery.js" %}"></script>
-    <script src="{% static "v2/js/metamask-approval.js" %}"></script>
+    <script src="{% static "v2/js/jquery.js" %}"></script>
```
Comment (octavioamu) [inline app/dashboard/templates/ens/base.html]: jquery.js doesn't live on that address...

**Comment by thelostone-mc** [comment]
Closing this out as this has been resolved ^_^
